api/spellcheck.py

```diff
@@ -1,59 +1,92 @@
-"""
-Module for spell check related code. Currently only contains aspell integration
-for the MVP, but may be extended to use AWS Kendra, custom dictionaries,
-consistency checking .etc.
-"""
-
-import subprocess
+import json
+import time
+import requests
+import os
 
 class Mistake():
     def __init__(self, word: str, location: int, suggestions: list[str]):
         self.word = word
         """
         The actual word that was misspelt.
         """
-        
+
         self.start = location
         """
         The character index at which the mistake starts.
         """
-        
+
         self.end = self.start + len(word)
         """
         The character index at which the mistake ends (exclusive).
         """
-        
+
         self.suggestions = suggestions
         """
-        The suggestions returned by aspell.
+        The suggestions returned by Azure Bing Spell Check.
         """
 
     def __repr__(self) -> str:
         return str(self.__dict__)
 
+
+def get_key():
```

Comment:
Praise: I love that yous added this function for if we change how to get the api key in the future.